tbconnect/models.py

```diff
@@ -93,7 +82,7 @@ class TBCheck(ExportModelOperationsMixin("tb-check"), models.Model):
     city_location = models.CharField(
         max_length=255, validators=[geographic_coordinate], null=True
     )
-    cough = models.CharField(max_length=13, choices=COUGH_CHOICES)
+    cough = models.BooleanField()
```

Comment:
Any idea what this migration does to existing data? I'm wondering if we need to map existing data somehow, or if it will just convert any string to `True`

Reply:
@rudigiesler I'm not sure, we'll find out in QA hehe, we haven't deployed to prod yet so not too worried.

Thanks for the review.